packages/dev/inspector/src/components/sceneExplorer/sceneExplorer.scss

```diff
@@ -436,13 +436,17 @@
             grid-template-columns: 1fr 20px 20px auto 5px;
             align-items: center;
 
-            .enableLight {
+            .visibi {

```

Comment:
Not sure it should have changed but might be fully wrong ??? all the rest is GG :-)